rotkehlchen/chain/evm/node_inquirer.py

```diff
@@ -460,13 +461,12 @@ def attempt_connect(
                         current_block = web3.eth.block_number  # pylint: disable=no-member
                         if isinstance(current_block, int) is False:  # Check for https://github.com/rotki/rotki/issues/6350  # TODO: Check if web3.py v6 has a check for this # noqa: E501
                             raise RemoteError(f'Found non-int current block:{current_block}')
-                        latest_block = self.query_highest_block()
                     except (requests.exceptions.RequestException, RemoteError) as e:
                         msg = f'Could not query latest block due to {e!s}'
                         log.warning(msg)
                         synchronized = False
                     else:
-                        synchronized, msg = _is_synchronized(current_block, latest_block)
+                        synchronized, msg = _is_synchronized(current_block, self.etherscan_block)
```

Comment:
add an assertion here that the etherscan_block is not 0. Assertions are removed for production